lib/ui/myplot/my_plot_child_item.dart

```diff
@@ -12,7 +12,6 @@ import 'package:flutter/material.dart';
 class MyPlotListItem {
   Widget buildListItem(CropEntity cropsData, Function goToDetail, Function goToStage) {
```

Comment:
lets introduce view models to remove the direct access to the model objects.
Maybe something like:
PlotListItemViewModel:
- title
- subtitle
- detailText.    (day 6)
- thumbnailURI
- progress     (0-1)